openassessment/xblock/ui_mixins/mfe/ora_config_serializer.py

```diff
@@ -221,3 +222,20 @@ def get_baseAssetUrl(self, block):
 
     def get_prompts(self, block):
         return [prompt["description"] for prompt in block.prompts]
+
+
+class GetBlockLearnerSubmissionDataSerializer(Serializer):
```

Comment:
Nit: naming? Does it need that many qualifiers?

```suggestion
class ResponseDataSerializer(Serializer):
```